_translator-files/po/de/Running-a-Server.po

```diff
@@ -445,219 +483,223 @@ msgid ""
 #: ../wiki/en/Running-a-Server.md:177
 msgid "_Note: The Jamulus service name in the `ExecStart` line needs to be the same as the `.service` file name you created when setting systemd to control your Jamulus Server. So in this example it would be `Jamulus-Server.service`_"
-msgstr ""
+msgstr "_Hinweis: Der Name des Jamulus-Dienstes in der \"ExecStart\"-Zeile muss mit dem Namen der \"service\"-Datei übereinstimmen, die du beim Einrichten von systemd zur Steuerung Ihres Jamulus-Servers erstellt hast. In diesem Beispiel wäre das also `Jamulus-Server.service`_"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:179
 msgid "Run `sudo systemctl daemon-reload` to register them for first use."
-msgstr ""
+msgstr "Führe `sudo systemctl daemon-reload` aus, um die `.service` Dateien für die erste Verwendung zu registrieren."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:181
 msgid "Now you can run these with the `systemctl` command, for example:"
-msgstr ""
+msgstr "Jetzt kannst du diese zum Beispiel mit dem Befehl `systemctl` ausführen:"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:183
 msgid "`sudo systemctl start jamulusTogglerec` (assuming you named your unit file `jamulusTogglerec.service`)"
-msgstr ""
+msgstr "sudo systemctl start jamulusTogglerec\" (vorausgesetzt, du hast deine Unit-Datei \"jamulusTogglerec.service\" genannt)"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:185
 msgid "You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs."
-msgstr ""
+msgstr "Du kannst das Ergebnis dieser Befehle sehen, wenn du `systemctl status jamulus` ausführst oder die Protokolle einsiehst."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:188
 msgid "## Servers on the desktop"
-msgstr ""
+msgstr "## Server auf dem Desktop"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:190
 msgid "Jamulus can be run in Server mode from the desktop. This gives you a graphical user interface to control most of the settings."
-msgstr ""
+msgstr "Jamulus kann im Server-Modus vom Desktop aus gestartet werden. Dadurch erhälst du eine grafische Benutzeroberfläche zur Steuerung der meisten Einstellungen."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:192
 #, no-wrap
 msgid "**Windows users** - Use the \"Jamulus Server\" icon in the Windows Start menu.\n"
-msgstr ""
+msgstr "**Windows-Benutzer** - Verwende das Symbol \"Jamulus Server\" im Windows-Startmenü.\n"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:193
 #, no-wrap
 msgid "**macOS users** - Double-click the \"Jamulus Server\" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).\n"
-msgstr ""
+msgstr "**MacOS-Benutzer** - Doppelklicke auf das \"Jamulus Server\"-Symbol in \"Programme\" (vorausgesetzt, du hast die Dateien der Installation gemäß [dieser Anleitung] (Installation-for-Macintosh) dort gespeichert).\n"
```

Comment:
Remove space in `[dieser Anleitung] (Installation-for-Macintosh)`

Reply:
Noted and corrected